### ISSUE TITLE
auto.sh local options removed ?

### ISSUE THREAD

**Issue by wiperpaul** [naworkcaj/bdvl#13]
the `-l` flag to indicate use local file for auto.sh isnt working for me. I see its been removed from current version. Now I get hung up trying to use a local file rather than a wget/curl.

Did you add a different functionality that I missed ?

`if [ "$OPTION" != "-u" ] && [ "$OPTION" != "-l" ]; then
    echo "$USAGE" && exit
fi
[ "$OPTION" == "-u" ] && USE_URL=1
[ "$OPTION" == "-l" ] && USE_LOCAL=1`

**Comment by naworkcaj** [comment]
I did remove the ability to use files available locally. I can't 100% remember why to be frank, I think it was something to do with just making the script easier to deploy. Come to think of it, I think I ran into a bug which resulted in me favouring removing the local option > the remote option.
To begin with, I thought it a good addition, so I must've had some reason for removing it. I thought I updated everything else accordingly too when I removed it.
